datalad_next/url_operations/ssh.py

```diff
@@ -16,23 +16,19 @@
     Queue,
 )
 from typing import (
-    Any,
     Dict,
     Generator,
     IO,
 )
 from urllib.parse import urlparse
 
-from datalad_next.runners import (
-    GeneratorMixIn,
-    NoCaptureGeneratorProtocol,
-    Protocol as RunnerProtocol,
-    StdOutCaptureGeneratorProtocol,
-    ThreadedRunner,
-    CommandError,
-)
+from more_itertools import side_effect
+
+from datalad_next.consts import COPY_BUFSIZE
+from datalad_next.runners.iter_subproc import iter_subproc
+from datalad_next.iterable_subprocess.iterable_subprocess import IterableSubprocessError
```

Comment:
This is an indication that we should import `IterableSubprocessError` in `datalad_next.runners.iter_subproc`, such that client code can grab these pieces, that belong together, from the same spot.

Reply:
Thanks